templates/teams.html

```diff
@@ -0,0 +1,38 @@
+{% extends "layout.html" %}
+{% block body %}
+
+  <div class="p-strip is-shallow is-bordered u-image-position" style="overflow: hidden;">
+    <div class="row u-equal-height u-vertically-center">
+      <div class="col-6">
+        <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+                <img class="p-heading-icon__img" src="{{ team.logo_url }}"alt="" />
+                  <h1 class="p-heading--one p-heading-icon__title">{{ team.name }}</h1>
+            </div>
+            <h2 class="p-heading--two">{{ team.subtitle }}</h2>
+            <p class="p-heading--four">{{ team.description }}</p>
+        </div>
+          <p>
+            <a class="p-link--external" href="{{ team.url }}">Learn more about {{ team.name }}</a>
+          </p>
+        </ul>
+      </div>
+      <div class="col-5 prefix-1">
+        <img src="{{ team.hero_url }}"
+        class="hero-image u-hidden--small u-image-position--bottom"
+        alt="" />
+      </div>
+    </div>
+  </div>
+  {% for post in posts %}
```

Comment:
is it possible to create a component with this piece of code, because it isn't the first time I see this code.